src/tox/config/parallel.py

```diff
@@ -45,9 +45,11 @@ def add_parallel_flags(parser):
     parser.add_argument(
         "-p",
         "--parallel",
+        nargs="?",
+        const="auto",
         dest="parallel",
         help="run tox environments in parallel, the argument controls limit: all,"
-        " auto - cpu count, some positive number, zero is turn off",
+        " auto or missing argument - cpu count, some positive number, 0 to turn off",
```

Comment:
I would append `(default: auto)` to the end of the help string, which is more consistent with other arguments. e.g.

```
--workdir PATH                   tox working directory (default: None)
```

Reply:
It would be incorrect to do this because the default is `0` which is also added automatically by argparse libary. There is a bit of confusion beween the two types of defaults: the default when opion is not mentioned which is 0, and the default value when the option is mentioned (auto), known as `const`.  Maybe in the future we may want to change default to be auto, but this change has serious implications and should be posponed till a major version change. Mainly, if you want parallel, you must add at least `-p` for the moment.

Reply:
ah, thanks, I was conflating the two.